chart/kubeapps/templates/tests/test-chartsvc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-chartsvc-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-chartsvc-test
+    image: {{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag }}
```

Comment:
should we add a chartTests.image value?

Reply:
I don't mind but wouldn't that add noise to the values.yaml? It would be a corner case if someone needs that flexibility

Reply:
My only concern is that using frontend's image could be dangerous, it would have a side-effect we might forget about if we change that image. I don't think it's too bad to put it in values.yaml, users may want to pull from a different registry.